recipe/meta.yaml

```diff
@@ -1,50 +1,102 @@
 {% set name = "polars" %}
-{% set version = "0.18.3" %}
+{% set version = "0.20.18" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/polars-{{ version }}.tar.gz
-    sha256: efc3f629fddb060dc90c2dbd575824c1d662f37294c18c769d28e2bc4d5a1413
+    sha256: 8a321cbdbb459e3c0cc1af2ce6ac930d0d3b5ccbeb2dd3e4237ad07d487fd290
 
 build:
   number: 0
-  skip: true  # [py<37]
-  skip: true  # [win and python_impl=="pypy"]
+  # no rust nightly compiler for s390x
+  skip: true  # [s390x]
+  # skipping windows because missing maturin >=1.3.2 and since win is not a priority for SF
+  skip: true  # [win]
 
 requirements:
   build:
-    - python                              # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-    - crossenv                            # [build_platform != target_platform]
-    - maturin >=0.14,<0.15                # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
-    - posix                               # [win]
+    - python
+    - posix   # [win]
     - cmake
-    - make                                # [unix]
+    - make    # [unix]
     - cargo-bundle-licenses
+    - pip     # [linux and x86_64]
   host:
     - python
     - pip
-    - maturin >=0.14,<0.15
+    - wheel
+    - maturin 1.5.1
```

Comment:
I did this because it's the host section and we want hard pinnings there. Not sure if our policies changed. If they did, we should change this in the linter too.

Reply:
I committed this and will push soon.